src/client.ts

```diff
@@ -0,0 +1,44 @@
+import { Action, Link } from '@siren-js/core';
+import { isRecord, isString } from '@siren-js/core/dist/util/type-guard';
+import fetch from 'cross-fetch';
+import ClientResponse from './response';
+import { toEntryList, toURLSearchParams } from './entry';
+
+export { default as ClientResponse } from './response';
+
+export default class Client {
```

Comment:
Probably need a way even in 0.1.0 to add headers for `follow()` and `submit()`.

Reply:
I didn't want the first PR to be too large. That is very likely what will come next. See [this project](https://github.com/siren-js/client/projects/12).